packages/abstract-core/src/core/manager.rs

```diff
@@ -199,28 +199,28 @@ pub struct ModuleVersionsResponse {
 
 #[cosmwasm_schema::cw_serde]
 pub struct ModuleAddressesResponse {
-    pub modules: Vec<(String, String)>,
+    pub modules: Vec<(String, Addr)>,
 }
 
 #[cosmwasm_schema::cw_serde]
 pub struct ConfigResponse {
     pub account_id: Uint64,
     pub owner: String,
```

Comment:
Owner `String`

Reply:
Implemented by https://github.com/Abstract-OS/contracts/pull/308. Changing it here would lead to conflicts.